### PR TITLE
Add default implementation

### PR DESCRIPTION
**Pull request by AsceticBear**
To make the runtime config for parachain easier. `type CallValidator = Everything;`

### CODE REVIEW

**Comment by AurevoirXavier** [comment]
Recommend to implement for (). There is another Everything in Substrate. Avoid using the same name.

**Review by HackFisher — APPROVED** [review]

File: primitives/message-dispatch/src/lib.rs [hunk 1]
```diff
@@ -177,3 +180,15 @@ pub trait CallValidate<AccountId, Origin, Call> {
 		call: &Call,
 	) -> Result<(), TransactionValidityError>;
 }
+
+/// CallValidate's default implementation, no additional validation
+pub enum Everything {}
```
[the remark this answers is not in the document]
Comment (AsceticBear) [reply primitives/message-dispatch/src/lib.rs:185]: Just follow the substrate style:

```rs
pub enum Everything {}
impl<T> Contains<T> for Everything {
	fn contains(_: &T) -> bool {
		true
	}
}
```
and https://www.reddit.com/r/rust/comments/91mgqk/empty_enum_vs_empry_struct/. `Enum` makes sense here, we don't need to instance this type in runtime.

**Comment by AsceticBear** [comment]
> Recommend to implement for (). There is another Everything in Substrate with different meaning. Avoid using the same name.

It took me more time to choose the name between `()` and `Everything` than edit the code. Indeed, `Everything` means allowing all calls to pass, and `()` seems like no validation at all.  So, maybe `Everything` is better in my view.

**Comment by HackFisher** [comment]
`Everything` LGTM if it is from different namespace, more readable than `()`

**Pull request merged**